src/sensorpush_ble/parser.py

```diff
@@ -93,19 +132,27 @@ def _start_update(self, service_info: BluetoothServiceInfo) -> None:
         local_name = service_info.name
         result = {}
         device_type = None
-        for match_name in LOCAL_NAMES:
-            if match_name in local_name:
-                device_type = match_name
+        is_ht1 = False
+
         if (
-            not device_type
-            and "ef090000-11d6-42ba-93b8-9dd7ec090ab0" in service_info.service_uuids
+            local_name == "s"
+            and "ef090000-11d6-42ba-93b8-9dd7ec090aa9" in service_info.service_uuids
```

Comment:
Let's make the uuids constants so they aren't magic numbers that we won't remember later